sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/Error.cs

```diff
@@ -6,7 +6,7 @@
 namespace Azure.DigitalTwins.Core
 {
     [CodeGenModel("Error")]
-    internal partial class Error
+    public partial class ErrorInformation
```

Comment:
Maybe I am just missing some context - why did this type name change? 

Should the file name change also to ErrorInformation.cs?

Reply:
In particular, does this have any impact on the existing API error message types?

Reply:
This discussion was taken offline to teams